modules/deploy/pages/deployment-option/cloud/managed-connectors/create-iceberg-sink-connector.adoc

```diff
@@ -27,18 +27,26 @@ To create the Iceberg Sink connector:
 . On the **Create Connector** page, specify the following required connector configuration options:
 +
 |===
-| Property | Description
+| Property name | Property key | Description
 
 | `Topics to export`
+| `topics`
 | Comma-separated list of the cluster topics you want to replicate.
 
+| `Topics regex`
+| `topics.regex`
+| Java regular expression of topics to replicate. For example: specify `.\*` to replicate all available topics in the cluster. Applicable only when *Use regular expressions* is selected.
```

Comment:
```suggestion
| Java regular expression of topics to replicate. For example: specify `.*` to replicate all available topics in the cluster. Applicable only when **Use regular expressions** is selected.
```

Reply:
Formatting issue with asciidoc here. to see how we handled this in another doc:
https://github.com/redpanda-data/docs/commit/32d077a3e1fd080c67c5cb2076bfc1cb456d1116